lib/livebook/runtime/definitions.ex

```diff
@@ -411,6 +416,26 @@ defmodule Livebook.Runtime.Definitions do
       Exqlite.query!(conn, "PRAGMA table_list", [])\
       """,
       packages: [kino_db, exqlite]
+    },
+    %{
+      type: :file_action,
+      file_types: [".xlsx",".xlsm"],
+      description: "Read sheets",
+      source: """
+      xlsx_file = Kino.FS.file_path("{{NAME}}")
+      {:ok, package} = XlsxReader.open(xlsx_file)
+
+      tabs =
+        for sheet <- XlsxReader.sheet_names(package) do
+          # Assume the first row contains column names
+          {:ok, [header | rows]} = XlsxReader.sheet(package, sheet)
```

Comment:
I've just tested with some real sheets, and got a problem here in a scenario that I think it's very common:

the file has several sheets with values, but one of the sheets is empty and it fails matching in this line.

how about doing something like below?

```elixir
tabs =
  for sheet <- XlsxReader.sheet_names(package),
      {:ok, rows} = XlsxReader.sheet(package, sheet),
      length(rows) > 1 do
    [header | rows] = rows

    maps =
      Enum.map(rows, fn row ->
        header
        |> Enum.zip(row)
        |> Map.new(fn
          {k, :expect_chars} -> {k, nil}
          kv -> kv
        end)
      end)

    {sheet, Kino.DataTable.new(maps)}
  end
```

Reply:
also added this `:expect_chars` handling, because in many cases when the value is empty, that's what I get instead from the library, not sure if it's a behaviour we should change in xlsx_reader cc @xavier

Reply:
@rodrigues what if we add two options to xlsx_reader? One to skip empty sheets and another to nillify expect_chars?

Reply:
that sounds good to me, `sheet_names/2` could get handle `skip_empty: true`.

about `:expect_chars`, it sounds to me like maybe it should be `nil` by default, but I don't know the reasoning for returning it today.

Reply:
@rodrigues would you open up issues/send PRs to `xlsx_reader`, please? :)

Reply:
yep I'll find some time to work on these two 👌